src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -127,26 +146,43 @@ else if (!settingsDir.isDirectory())
      * methods defined here to allow default handling of builtin options.
      *
      * @param options Options to parse.
+     * @return True if {@code options} was parsed, False otherwise.
      */
-    protected void parseOption(final String[] options) {
-        if (options[0].equals(SETTINGS_DIR_OPTION))
+    protected boolean parseOption(final String[] options) {
+
+        boolean hasParsedOption = false;
```

Comment:
I think that it will be cleaner if this variable is removed, each if/else-if clause is returning true and at the end it returns false. But it is just a matter of taste...